draft-ietf-core-dns-over-coap.md

```diff
@@ -75,9 +72,9 @@ To prevent resource requirements of DTLS or TLS on top of UDP (e.g.,
 introduced by DNS over QUIC {{?RFC9250}}), DoC allows
 for lightweight end-to-end payload encryption based on OSCORE.
 
-~~~ drawing
+~~~ aasvg
```

Comment:
Is there some `aasvg` way to have the dotted/dashed arrow on the right-hand side of the graphic parsed correctly?

Reply:
It looked nice enough for me.
I'm not aware about dashed/dotted line support in aasvg.
Not sure my attempt at a8c912e is an improvement.

Reply:
IMHO it does. At least the arrow tips match now and the line isn't weirdly misaligned.